extensions/adobe/experience/decisioning/decisionevent.example.1.json

```diff
@@ -2,62 +2,55 @@
   "@id": "https://data.adobe.io/entities/propositions/3cc33a7e-13ca-4b19-b25d-c816eff9a70a",
   "xdm:timestamp": "2019-01-26T15:52:25+00:00",
   "xdm:identityMap": {
-    "ECID": [
-      { "xdm:id": "https://data.adobe.io/entities/identity/49312748749132" }
-    ]
-  },
-  "xdm:device": {
-    "xdm:typeID": "IMac",
-    "xdm:typeIDService": "https://ns.adobe.com/xdm/external/adobecampaign",
-    "xdm:type": "desktop",
-    "xdm:manufacturer": "Apple"
-  },
-  "xdm:placeContext": {
-    "xdm:localTime": "2017-09-26T15:52:25+13:00",
-    "xdm:geo": {
-      "@id": "https://data.adobe.io/entities/geo/tokyo",
-      "xdm:countryCode": "JP",
-      "xdm:stateProvince": "JP-13",
-      "xdm:city": "Tōkyō",
-      "xdm:postalCode": "141-0032",
-      "schema:latitude": 35.6185,
-      "schema:longitude": 139.73237
-    }
-  },
-  "xdm:web": {
-    "xdm:webPageView": {
-      "xdm:URL": "https://www.example.com"
-    },
-    "xdm:webReferrer": {
-      "xdm:URL": "https://www.examplereferrer.com/",
-      "xdm:domain": "examplereferrer.com"
-    }
```

Comment:
Remove properties from the example that aren't directly defined by the decision event schema